### PR TITLE
Search style

### PR DESCRIPTION
**Pull request by stellakunzang**
Fixed some front end bugs like the form fields and buttons being weird/not uniform. 
Formatted table which will eventually be used for search results, but is currently on All Obituaries page. 
Added a sentence indicating that someone died of COVID instead of just displaying true or false.

### CODE REVIEW

File: app/assets/stylesheets/application.scss [hunk 1]
```diff
@@ -69,14 +89,47 @@ body {
   transition: all 0.2s;
 }
 
+.big-checkbox {width: 20px; height: 20px;}
```
Comment (stellakunzang) [inline app/assets/stylesheets/application.scss:92]: the covid checkbox on the form was too small!

File: app/assets/stylesheets/application.scss [hunk 2]
```diff
@@ -69,14 +89,47 @@ body {
...
+
+.btn-submit {
+  font-size: 80%;
+  border-radius: 5rem;
+  letter-spacing: .1rem;
+  font-weight: bold;
+  padding: 1rem;
+  transition: all 0.2s;
+}
```
Comment (stellakunzang) [inline app/assets/stylesheets/application.scss:101]: we now have a general button class with the shape and styling as on the signin signup forms

File: app/assets/stylesheets/application.scss [hunk 3]
```diff
@@ -69,14 +89,47 @@ body {
...
+  width: 100%;
+  margin-bottom: 0;
+  line-height: 1.5;
+  padding: 0.5rem;
+}
+
+.table-striped>tbody>tr:nth-child(odd)>td,
+.table-striped>tbody>tr:nth-child(odd)>th {
+   background-color: #edeeea; // Choose your own color here
+ }
```
Comment (stellakunzang) [inline app/assets/stylesheets/application.scss:125]: customizes the color of the alternating stripes on the table.

File: app/models/obituary.rb [hunk 4]
```diff
@@ -5,6 +5,6 @@ class Obituary < ApplicationRecord
 
   private
   def default_image
-    self.image_url ||= 'https://image.shutterstock.com/image-vector/ui-image-placeholder-wireframes-apps-260nw-1037719204.jpg'
+    self.image_url ||= '/thumbnail_logo.png'
```
Comment (stellakunzang) [inline app/models/obituary.rb:8]: Thumbnail logo (just the newspaper) is now our default image for obituaries!

File: app/views/obituaries/_form.html.erb [hunk 5]
```diff
@@ -36,23 +36,25 @@
...
 
           <div class="form-group" id="age-select">
             <%= f.label :age, class: 'col-form-label-lg'%>
             <%= f.select :age, options_for_select((0..150), 100), { }, {class: 'custom-select' } %>
           </div>
 
-          <div class="form-check" id="covid-select">
-            <%= f.label :covid, "COVID-19", class: 'col-form-label-lg' %>
-            <%= f.check_box :covid  %>
+          <div class="form-check row" id="covid-select">
```
Comment (stellakunzang) [inline app/views/obituaries/_form.html.erb:51]: turns out the word "row" is really important for getting everything to line up!

File: app/views/obituaries/index.html.erb [hunk 6]
```diff
@@ -1,26 +1,26 @@
...
+      </thead>
+      <tbody style= "background-color:white;">
+        <% @obituaries.each do |obituary| %>
+          <tr id="obit-<%= obituary.id %>">
+            <td><%= link_to image_tag(obituary.image_url, size: "50x50"), "/obituaries/#{obituary.id}" %></td>
+            <td class="align-middle pl-3"><%= obituary.first_name + ' ' + obituary.last_name%></td>
+            <td class="align-middle pl-3"><%= obituary.state %></td>
+            <td class="align-middle pl-3"><%= obituary.age %></td>
+            <td class="align-middle pl-3"><%= "2020" %></td>
+            <td class="align-middle pl-3"><%= "ObituaryHub Community" %></td>
```
Comment (stellakunzang) [inline app/views/obituaries/index.html.erb:22]: Hardcoded for display purposes!

File: app/models/obituary.rb (hunk 4 above)
Comment (jpc20) [reply app/models/obituary.rb:8]: Great idea!

File: app/assets/stylesheets/application.scss (hunk 3 above)
Comment (jpc20) [reply app/assets/stylesheets/application.scss:125]: Cool. I can't wait to check this out.

File: app/assets/stylesheets/application.scss (hunk 2 above)
Comment (EricLarson2020) [reply app/assets/stylesheets/application.scss:101]: Nice Work!

File: app/assets/stylesheets/application.scss (hunk 3 above)
Comment (EricLarson2020) [reply app/assets/stylesheets/application.scss:125]: Nice Work!

File: app/models/obituary.rb (hunk 4 above)
Comment (EricLarson2020) [reply app/models/obituary.rb:8]: Love it!

File: app/views/obituaries/_form.html.erb (hunk 5 above)
Comment (EricLarson2020) [reply app/views/obituaries/_form.html.erb:51]: good to know

File: app/views/obituaries/index.html.erb (hunk 6 above)
Comment (EricLarson2020) [reply app/views/obituaries/index.html.erb:22]: Great work Styling

File: app/assets/stylesheets/application.scss (hunk 2 above)
Comment (sagemlee) [reply app/assets/stylesheets/application.scss:101]: love it, I appreciate you going on the search for the buttons/roundness!

File: app/models/obituary.rb (hunk 4 above)
Comment (sagemlee) [reply app/models/obituary.rb:8]: Perfect

File: app/views/obituaries/_form.html.erb (hunk 5 above)
Comment (sagemlee) [reply app/views/obituaries/_form.html.erb:51]: Found this out last night too

File: app/views/obituaries/index.html.erb (hunk 6 above)
Comment (sagemlee) [reply app/views/obituaries/index.html.erb:22]: " + ' '  + " is a much better way to write in a space then the huge gap that I left, will be using this in the future!

**Comment by sagemlee** [comment]
This all looks awesome Stella, great job!

**Pull request merged**